packages/terra-heading/src/terra-dev-site/doc/example/colors.css

```diff
@@ -0,0 +1,22 @@
+:root {
```

Comment:
Should these styles be locally scoped?

Reply:
If we don't use `:root` our linter complains that the variables are defined outside of `:root`.

Reply:
Since these are base css, we can't use the `:local` scope. I was concerned that the goal of these was to be as simple as possible for the examples, so turning them into `colors.modules.scss` might be more then necassary.

Reply:
Yeah, I'm fine keeping these as-is and not running them through CSS Modules syntax.